### PR TITLE
Update gloss-marathi.ldf

### PR DESCRIPTION
**Pull request by NiranjanTambe**

### CODE REVIEW

File: tex/gloss-marathi.ldf [hunk 1]
```diff
@@ -1,91 +1,97 @@
...
-  \xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
+\xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
 I will proceed with the compilation, but\MessageBreak
 the output is not guaranteed to be correct\MessageBreak
 and may look very wrong.}
 \fi
 \RequirePackage{devanagaridigits}
 
 \PolyglossiaSetup{marathi}{
-  bcp47=mr,
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: please do not remove existing tags such as `bcp47` or `localnumeral`. I think you should not touch `\PolyglossiaSetup` at all-

File: tex/gloss-marathi.ldf [hunk 2]
```diff
@@ -1,91 +1,97 @@
...
+langtag=MAR,
+hyphennames={marathi},
+hyphenmins={2,2},%CHECK
+fontsetup=true,
+%TODO nouppercase=true,
+%TODO localnumber=marathinumber
 }
 
-% BCP-47 compliant aliases
-\setlanguagealias*{marathi}{mr}
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: Please do not remove this! This is a new feature.

File: tex/gloss-marathi.ldf [hunk 3]
```diff
@@ -1,91 +1,97 @@
...
-
+\renewcommand{\baselinestretch}{1.2}
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: remove this and move it to `\extras`

File: tex/gloss-marathi.ldf [hunk 4]
```diff
@@ -1,91 +1,97 @@
...
 \newif\ifmarathi@devanagari@numerals
 \marathi@devanagari@numeralstrue
 
 \define@key{marathi}{numerals}[Devanagari]{%
-  \def\@tmpa{#1}%
-  \ifx\@tmpa\tmp@western
-    \marathi@devanagari@numeralsfalse
-  \fi}
-
-\newcommand{\marathinumerals}[2]{\marathinumber{#2}}
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: Please do not touch this block

File: tex/gloss-marathi.ldf [hunk 5]
```diff
@@ -1,91 +1,97 @@
...
+\ifx\@tmpa\tmp@western
+\marathi@devanagari@numeralsfalse
+\fi}
 
 \def\marathinumber#1{%
-  \ifmarathi@devanagari@numerals
-    \devanagaridigits{\number#1}%
-  \else
-    \number#1%
-  \fi}
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: Also, why do you revert the indentation here?

File: tex/gloss-marathi.ldf [hunk 6]
```diff
@@ -1,91 +1,97 @@
...
+\def\pagename{पृष्ठ}%
+\def\seename{पाहा}%
+\def\alsoname{हेदेखील पाहा}%
+\def\enclname{समाविष्ट}%
+\def\ccname{सी.सी.}%
+\def\headtoname{प्रती}%
+\def\proofname{सिद्धता}%
+\def\glossaryname{संक्षेपसूची}%
+\def\appendixname{परिशिष्ट}
 }
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: Please keep the indentation as in the original

File: tex/gloss-marathi.ldf [hunk 7]
```diff
@@ -1,91 +1,97 @@
...
+जून\or
+जुलै\or
+ऑगस्ट\or
+सप्टेंबर\or
+ऑक्टोबर\or
+नोव्हेंबर\or
+डिसेंबर\fi
+}%
+\def\today{\marathinumber\day\space\marathimonth\space\marathinumber\year}%
 }
```
Comment (jspitz) [inline tex/gloss-marathi.ldf]: Dito.

**Comment by jspitz** [comment]
To resolve the issues above, I suggest you start from scratch using `gloss-marathi.ldf` from master.

**Comment by NiranjanTambe** [comment]
I am really sorry. I picked up the `ldf` file from my texlive installation. As it was an old version, it changed a lot of new things. That was not intended. I'll correct my mistake soon and edit it the correct way.

**Comment by jspitz** [comment]
No problem! Could you please also check whether the warning about “Marathi not being supported in LuaTeX” is still valid? I.e., does the LuaTeX output look correct nowadays?

File: tex/gloss-marathi.ldf (hunk 6 above)
Comment (NiranjanTambe) [reply tex/gloss-marathi.ldf]: Is it right now?

**Comment by jspitz** [comment]
The new version looked much better (except that the caption changes were missing). Why did you close this PR?

**Pull request closed without merging**